### PR TITLE
Remove WS constants

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Remove WS constants and replace theirs usages with WS enum values.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2212)

### CODE REVIEW

**Comment by JeremyKuhne** [comment]
Can you please resolve conflicts? Thanks for you contribution!

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs [hunk 1]
```diff
@@ -2382,7 +2382,7 @@ protected override CreateParams CreateParams
                     {
                         CreateParams cp = base.CreateParams;
                         cp.ExStyle |= (int)User32.WS_EX.TOOLWINDOW;
-                        cp.Style |= NativeMethods.WS_POPUP | NativeMethods.WS_BORDER;
+                        cp.Style |= unchecked((int)User32.WS.POPUP) | (int)User32.WS.BORDER;
```
[the remark this answers is not in the document]
Comment (gpetrou) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs]: There is an "Overflow in constant value computation" without it. The original const had it as well.

[the remark this answers is not in the document]
Comment (RussKie) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs]: Thanks, I missed it.

Comment (RussKie) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs]: @gpetrou I think it is slightly less verbose and easier on an eye to cast once:
```suggestion
                        cp.Style |= unchecked((int)(User32.WS.POPUP | User32.WS.BORDER);
```
The same applies to other cases

@JeremyKuhne a risk of an error in this way is probably higher than the old way, as now the caller must remember to call `unchecked`.

Comment (JeremyKuhne) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs]: You don't need to use unchecked, and in fact, shouldn't. All runtime casts are unchecked, only constants are checked (i.e. compile time only).

Comment (RussKie) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs]: It doesn't work without `unchecked` though
image

So I guess my worry was unfounded, the compiler will err and warn about the lack of `unchecked`.

**Comment by RussKie** [comment]
Please rewrite `(int)User32.WS.xyz) | (int)User32.WS.abc` as `(int)(User32.WS.xyz | User32.WS.abc)` and it will be good to go.

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs (hunk 1 above)
Comment (gpetrou) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs]: I have updated the code based on your suggestions.

**Pull request merged**